app/models/channel.rb

```diff
@@ -271,6 +274,25 @@ def register_channel_for_promo
     Promo::RegisterChannelForPromoJob.perform_now(channel_id: id, attempt_count: 0)
   end
 
+  def update_site_banner_lookup!(skip_site_banner_info_lookup: false)
+    site_banner_lookup = SiteBannerLookup.find_or_initialize_by(
+      channel_identifier: details&.channel_identifier || details.brave_publisher_id,
```

Comment:
I think we want this strictly to be 
```suggestion
      channel_identifier: details&.channel_identifier,
```

if there's a problem with the  channel_identifier not working for `SiteChannelDetails` then we should probably fix that rather than have this `||`?